tasmoadmin/composer.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "reloxx13/tasmoadmin",
+    "type": "project",
+    "license": "GPL3",
+    "require": {
+        "php": ">=7.4",
```

Comment:
This would be a big change as before it was any version. 

The hassio addon is using PHP 8 and PHP 7.3 is EOL so this is the only PHP 7 series that is still supported.

See: https://www.php.net/supported-versions.php